diff --git a/external/kokkos-kernels/KokkosBatched_Dot_Internal.hpp b/external/kokkos-kernels/KokkosBatched_Dot_Internal.hpp
index e737434..9552fa0 100644
--- a/external/kokkos-kernels/KokkosBatched_Dot_Internal.hpp
+++ b/external/kokkos-kernels/KokkosBatched_Dot_Internal.hpp
@@ -5,6 +5,8 @@
 
 #include "KokkosBatched_Util.hpp"
 
+#define KOKKOS_IMPL_DO_NOT_USE_PRINTF(...) ::printf(__VA_ARGS__)
+
 namespace KokkosBatched {
 
 ///
diff --git a/external/kokkos-kernels/KokkosBatched_Util.hpp b/external/kokkos-kernels/KokkosBatched_Util.hpp
index 46b97ee..ae71a60 100644
--- a/external/kokkos-kernels/KokkosBatched_Util.hpp
+++ b/external/kokkos-kernels/KokkosBatched_Util.hpp
@@ -772,8 +772,8 @@ KOKKOS_INLINE_FUNCTION auto subview_wrapper(ViewType v, IdxType1 i1,
 }
 template <class ViewType, class IdxType1>
 KOKKOS_INLINE_FUNCTION auto subview_wrapper(ViewType v, IdxType1 i1,
-                                            Kokkos::Impl::ALL_t i2,
-                                            Kokkos::Impl::ALL_t i3,
+                                            Kokkos::ALL_t i2,
+                                            Kokkos::ALL_t i3,
                                             const BatchLayout::Left &layout_tag,
                                             const Trans::Transpose) {
   auto sv_nt = subview_wrapper(v, i1, i3, i2, layout_tag);
@@ -805,7 +805,7 @@ KOKKOS_INLINE_FUNCTION auto subview_wrapper(
 }
 template <class ViewType, class IdxType1>
 KOKKOS_INLINE_FUNCTION auto subview_wrapper(
-    ViewType v, IdxType1 i1, Kokkos::Impl::ALL_t i2, Kokkos::Impl::ALL_t i3,
+    ViewType v, IdxType1 i1, Kokkos::ALL_t i2, Kokkos::ALL_t i3,
     const BatchLayout::Right &layout_tag, const Trans::Transpose &) {
   auto sv_nt = subview_wrapper(v, i1, i3, i2, layout_tag);
 
